sjb/config/test_cases/test_pull_request_openshift_ansible_extended_conformance_gce_37.yml

```diff
@@ -0,0 +1,15 @@
+---
+parent: 'common/test_cases/origin_release_install_gce.yml'
+overrides:
+  sync:
+    - "openshift,origin=master"
+    - "openshift,aos-cd-jobs=master"
+    - "openshift,image-registry=master"
+    - "openshift,kubernetes-metrics-server=master"
```

Comment:
master for 3.7?

Reply:
i'll just remove them - 3.7 doesn't use these at all.

Reply:
Actually I need to sync something or the job will fail.

Reply:
These repos wouldn't build before 3.9